spec/foodzone/venues/address_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../../../app/foodzone/venues/address'
+
+describe Venues::Address do
+  let(:address) { described_class.from_str(address_str) }
+  let(:address_str) { 'My place, #02, Hood' }
+
+  describe '::from_str' do
+    it 'extract street' do
+      expect(address.street).to eql 'My place'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.